### DISCUSSION TITLE
WiFi gone

### DISCUSSION THREAD

**Discussion by hartsie** [OnionUI/Onion#1267] [Support]
### Checklist

- [X] I am reporting a bug in the Onion OS - Any issues with the closed source frontend ([MainUI](https://github.com/OnionUI/MainUI-issues)), or [RetroArch](https://github.com/libretro/RetroArch) or any of the cores should be directed to their respective repositories
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

4.2.0 Beta

### Miyoo Firmware Version

202305052130

### Provide a clear and concise description of the issue

I placed my card in a MM v4 and back into the MM+ and now my WiFi option is gone from settings. 

### Steps to Reproduce

Please provide detailed steps that will help us replicate and investigate your issue

see above

### Provide any additional context or information that may help us investigate your issue

None

**Comment by Aemiii91** [comment]
Earlier beta versions could sometimes have issues when swapping. Please try updating.

**Reply by hartsie** [reply]
Thank you!
